src/backend/marsha/core/tests/test_api_document.py

```diff
@@ -77,13 +78,15 @@ def test_api_document_fetch_instructor(self):
             content,
             {
                 "active_stamp": "1533686400",
+                "extension": "pdf",
```

Comment:
_get_extension_string?

Reply:
> So the `_get_extension` method returned value and the stored file extension are not equal right? (`.pdf` _vs_ `pdf`)

Right. The stored value does not have the leading dot. The method `_get_extension` return it with the leading dot **or** an empty string if there is no extension (because yes, it's a valid case).

Reply:
I'm talking about code consistency: semantically speaking this is not the case.

Reply:
ok